wizard/models.py

```diff
@@ -587,7 +610,10 @@ def clean(self):
         if might_be_ready:
             try:
                 if self.input.rows.count != self.target.rows.count:
-                    raise ValidationError('The number of rows in the input and target do not match')
+                    self.is_ready = True
+                    archive_warning = "Missing data detected. Proceed with caution."
+                    print(archive_warning)
```

Comment:
again, not sure we should be printing this? Maybe `logger.debug` or something? Or return it to user?